helper/hex/hex_test.go

```diff
@@ -0,0 +1,42 @@
+package hex
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestEncodingUint64(t *testing.T) {
```

Comment:
Why not have this test simply generate a random number array (with 0 included), and test it out that way?

Reply:
fixed lint 70e04dfc8686d02178eb0868d3bf78e3f7d54f4f

Reply:
What am I getting with that test, what random number will show me that something is not working, why would I have the necessary generation if it doesn't test anything?  
This subset of numbers contains min, max, and few in between. 
If you guys want this we can add it. Don't see any value in adding a generator method locally. Maybe if we add a global util function that we can use from all unit tests it will make sense :0